octavia/common/rpc.py

```diff
@@ -74,3 +74,16 @@ def get_notifier(service=None, host=None, publisher_id=None):
 
 def create_transport(url):
     return messaging.get_rpc_transport(cfg.CONF, url=url)
+
+
+def get_notification_listener(targets, endpoints, serializer=None):
+    if NOTIFICATION_TRANSPORT is None:
+        raise AssertionError("'NOTIFICATION_TRANSPORT' must not be None")
```

Comment:
Simpler: `assert NOTIFICATION_TRANSPORT, "NOTIFICATION_TRANSPORT must not be None"`